1209/6809/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: pid
+title: Doppelganger
+owner: TachRadio
+license: BSD-2-Clause
+site: https://thewired.doesntexist.org
+source: https://github.com/NilujePerchut/doppelganger_hardware
+	https://github.com/NilujePerchut/doppelganger_soft
```

Comment:
I don't think the frontporch will handle both of these at once. Can you link to the software here and link to the hardware in the body text below.

I'm planning to try and add a few more source variants to differentiate.